projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  (click)="closeOffCanvas()"
+  [@fade]="opened ? 'in' : 'out'"
+  class="go-off-canvas-backdrop"
+  tabindex="-1"
+>
+</div>
+<div
+  [@offCanvas]="opened ? 'slideIn' : 'slideOut'"
+  class="go-off-canvas"
+  role="dialog"
+>
+  <div class="go-off-canvas__header">
+    <button
```

Comment:
Should this close button be a footer instead of a header?

Reply:
It could be, I thought maybe we would want to have additional options in the footer. This will be a static way to close out the off canvas across all instances.

Reply:
Ya I agree that we should have the static option to close out the off canvas, but I'm not sure the header is where we would want that. I know currently (which may not be what we want in the future) there are a lot of places where we have a search or a title in the header. I'm just thinking about if a close button above that title is something that would look alright?